### PR TITLE
Updated for R5C

### PR DESCRIPTION
**Pull request by ctech4285**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +33,16 @@ The build outputs will be available in the directory `build/`.
 
 ### Useful info
 
-* Do not cycle the EPM and you cannot get zapped.
+* If you do not cycle the EPM you cannot get zapped.
 * Don't zap your computer it will not survive.
 * HV section to avoid touching during debug, see fig. 1.
-* Solder Jumper settings see fig. 1.
+* Solder Jumper settings see fig. 1 and 2
```
Comment (pavel-kirienko) [inline README.md]: Period

File: README.md [hunk 2]
```diff
@@ -33,13 +33,16 @@ The build outputs will be available in the directory `build/`.
...
 
+#### V3 Rev 4B and earlyer
 ![Figure 1](annotation.jpg "Figure 1")
 
+#### V3 Rev 5C
+![Figure 1](annotationR5C.jpg "Figure 1")
 ### Via UART
```
Comment (pavel-kirienko) [inline README.md]: Missing newline

File: README.md [hunk 3]
```diff
@@ -64,15 +67,16 @@ Parameter       | Value
 Baud rate       | 115200
 Interface       | ISP
 Oscilator       | 12 MHz
-File            | Use the `.hex` file in the build output directory**
+File            | Prebuild `.hex` available [here](https://files.zubax.com/products/com.zubax.opengrab_epm_v3/) **
```
Comment (pavel-kirienko) [inline README.md]: Prebuilt

File: README.md [hunk 4]
```diff
@@ -64,15 +67,16 @@ Parameter       | Value
...
 
 1. Connect the USB-UART adapter.
 2. Close J3 to select serial bootloader.
 3. Close J4 to force the LPC to start the bootloader.
 4. Power up the board.
 5. Run the flashing tool.
 
-* Prebuilt `.hex` file is available [here](https://files.zubax.com/products/com.zubax.opengrab_epm_v3/)
+* ** `.hex` is also created in the build output directory `firmware/build`
+
```
Comment (pavel-kirienko) [inline README.md]: Extra newline; the bullet points are not needed here.

File: README.md (hunk 1 above)
Comment (ctech4285) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (ctech4285) [inline README.md]: done

File: README.md (hunk 4 above)
Comment (ctech4285) [inline README.md]: done

**Pull request merged**